src/components/modes/Send.vue

```diff
@@ -60,6 +60,8 @@ export default defineComponent({
           },
         };
       }
+
+      console.log(this.entity);
```

Comment:
Do you usually keep console.logs for deploy and in git?

Reply:
No i don’t , will remove that